mqtt/mqtt-broker/src/configuration.rs

```diff
@@ -0,0 +1,322 @@
+use std::convert::From;
```

Comment:
Why does this diff show? I'm guessing it was from a change ported to master (i.e. rust stable / clippy fixes). But then why isn't this also in iiot now?

Reply:
Alright so it seems this was later moved so we will have to delete it. Doing so will then fix the checkin gate.